### PR TITLE
Card Component

### PR DESCRIPTION
**Pull request by shagunbagrecha**
Variants of Card :
 - Basic Vertical Card
 - Basic Horizontal Card
 - Card with a Badge
 - Card with Dismiss
 - Card with Text Overlay
 - Text Only Cards
 - Card with Shadow

Preview: https://deploy-preview-5--vivid-ui.netlify.app/

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
looks good

File: components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/* All Cards */
+
+.card, .card-badge{
+	border-radius: 0.5rem;
+	font-family: var(--font-secondary);
+}
+.card{
+	position:relative;
```
Comment (manojasarsa) [inline components/Card/card.css]: You can use "beautify file extension" to beautify your file. So, that consistent spacing and indentation can be achieved.

File: components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+}
+
+/* Card with text-overlay */
+.card-text-overlay{
+	position: absolute;
+	border-radius: 0.5rem;
+	background-color: var(--clr-gray);
+	color: var(--clr-white);	
+	top: 0;
+	bottom:0;
```
Comment (manojasarsa) [inline components/Card/card.css]: space needed here as well

File: components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+	right: 0;
+	left:0;
```
Comment (manojasarsa) [inline components/Card/card.css]: same spacing

**Review by manojasarsa — APPROVED** [review]
Rest the card components looks pretty good. Nice work!

File: components/Card/card.css [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+/* Card with Shadow */
+.card-shadow{
+	/* box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px; */
+	box-shadow: rgba(50, 50, 93, 0.25) 0px 30px 60px -12px inset, rgba(0, 0, 0, 0.3) 0px 18px 36px -18px inset;
+}
```
Comment (siddhigate) [inline components/Card/card.css]: - You can remove this comment to make your code cleaner.
- You can use CSS variables for box-shadow values
- Apart from the points mentioned by @manojasarsa , everything looks good🙌

**Review by siddhigate — APPROVED** [review]

**Pull request merged**